lib/commands/notes.js

```diff
@@ -0,0 +1,52 @@
+import { SlashCommandBuilder } from '@discordjs/builders'
+import notes from '../notes.js'
+
+export default {
+  ephemeral: true,
+  data: new SlashCommandBuilder()
+    .setName('note')
+    .setDescription('Manage shared notes for users')
+    .addSubcommand(sub => sub
+      .setName('get')
+      .setDescription('Get the shared note for a user')
+      .addUserOption(option => option
+        .setName('user')
+        .setDescription('The user to get the note for')
+        .setRequired(true))
+    )
+    .addSubcommand(sub => sub
+      .setName('set')
+      .setDescription('Set the shared note for a user')
+      .addUserOption(option => option
+        .setName('user')
+        .setDescription('The user to set the note for')
+        .setRequired(true))
+      .addStringOption(option => option
+        .setName('value')
+        .setDescription('New note for the user')
+        .setRequired(true))
+    )
+    .addSubcommand(sub => sub
+      .setName('del')
+      .setDescription('Delete the shared note for a user')
+      .addUserOption(option => option
+        .setName('user')
+        .setDescription('The user to delete the note from')
+        .setRequired(true))
+    ),
+  permissions: ['moderator', 'helper'],
+  async execute (interaction) {
+    const user = interaction.options.getUser('user')
+
+    switch (interaction.options.getSubcommand()) {
+      case 'get':
+        return interaction.followUp(await notes.getNote(user))
+      case 'set': {
```

Comment:
you dont need to use block here

Reply:
If I don't, it complains about assigning a variable inside the case.

Reply:
oh hmm, alright